Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Sliders/PinchSlider.cs

```diff
@@ -53,8 +85,35 @@ public float SliderValue
         [Tooltip("The axis the slider moves along")]
         [SerializeField]
         private SliderAxis sliderAxis = SliderAxis.XAxis;
+        public SliderAxis CurrentSliderAxis
+        {
+            get { return sliderAxis; }
+            set
+            {
+                sliderAxis = value;
+                UpdateVisualsOrientation();
+            }
+        }
+
+        private SliderAxis? previousSliderAxis = null;
+        public SliderAxis PreviousSliderAxis
+        {
+            get
+            {
+                if (previousSliderAxis == null)
+                {
+                    previousSliderAxis = CurrentSliderAxis;
+                }
+                return previousSliderAxis.Value;
+            }
+            set
+            {
+                previousSliderAxis = value;
+            }
+        }
+
         [Serializable]
-        private enum SliderAxis
+        public enum SliderAxis
```

Comment:
If possible, it'd be great to move this newly `public` enum [into its own file](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#public-classes-structs-and-enums-should-all-go-in-their-own-files).